packages/docmanager-extension/src/index.ts

```diff
@@ -439,9 +439,9 @@ function addCommands(app: JupyterLab, docManager: IDocumentManager, palette: ICo
         return;
```

Comment:
This one isn't your fault, but while we are at it, can we make the label for this command in capital case ("Show in File Browser") for consistency with the other menu labels?

Reply:
Though it's true that my own preference is to have at most one capital letter in a title, I honestly hadn't noticed this until you pointed it out. I'd be glad to make it match the existing style

Reply:
Yeah, I personally don't have a strong preference, except for consistency. We are trying to roughly follow the Apple style guidelines for menu items, which uses capital case.